pkg/operator/cassandra/controller/rbac.go

```diff
@@ -0,0 +1,132 @@
+package controller
+
+import (
+	cassandrav1alpha1 "github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1"
+	"github.com/rook/rook/pkg/operator/cassandra/controller/util"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SyncMemberRole checks if the member Role exists and creates it if it doesn't
```

Comment:
let's think though further, or at least justify further, if we can avoid creating/manipulating RBAC permissions at runtime.  maybe @travisn can help with that.